fw.lua

```diff
@@ -296,6 +300,16 @@ local function _log_event(self, request_client, request_uri, rule, match)
 		t.rule.var = rule.var
 	end
 
+	if self._event_log_save_post_data then
+		t.post_body = collections["REQUEST_BODY"]
+	end
+
+	if (table.getn(self._event_log_ngx_vars) ~= 0) then
```

Comment:
This might be unnecessary; if self._event_log_ngx_vars was initialized to an empty table, and not changed via set_option, wouldn't the ipairs() call just gloss over the empty table?